docs/getting-started.md

```diff
@@ -17,22 +17,46 @@ instance using Ubuntu 18.04 on EC2. Firecracker uses
 [KVM](https://www.linux-kvm.org) and needs read/write access that can be
 granted as shown below:
 
-```
-sudo setfacl -m u:${USER}:rw /dev/kvm
-```
+ ```
+ sudo setfacl -m u:${USER}:rw /dev/kvm
+ ```
 
 The generic requirements are explained below:
 
 - **Linux 4.14+**
 
   Firecracker currently supports physical Linux x86_64 hosts, running kernel
-  version 4.14 or later.
+  version 4.14 or later. Please check first if your Linux host supports vitualization
+  running as root the following command:
+  
+  ```
+  egrep '(vmx|svm)' --color=always /proc/cpuinfo
+  ```
+  you should see the following answer (number of flags depends of the number of cores on the CPU):
+  
+  ```
+  flags		: fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ht syscall nx   mmxext fxsr_opt pdpe1gb rdtscp lm 3dnowext 3dnow constant_tsc rep_good nopl nonstop_tsc cpuid extd_apicid pni monitor cx16   popcnt lahf_lm cmp_legacy svm extapic cr8_legacy abm sse4a misalignsse 3dnowprefetch osvw ibs skinit wdt nodeid_msr         hw_pstate vmmcall npt lbrv svm_lock nrip_save
+  flags		: fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ht syscall nx   mmxext fxsr_opt pdpe1gb rdtscp lm 3dnowext 3dnow constant_tsc rep_good nopl nonstop_tsc cpuid extd_apicid pni monitor cx16   popcnt lahf_lm cmp_legacy svm extapic cr8_legacy abm sse4a misalignsse 3dnowprefetch osvw ibs skinit wdt nodeid_msr         hw_pstate vmmcall npt lbrv svm_lock nrip_save
+
```

Comment:
It might be nice to show an example from Intel and ARM CPUs.

Reply:
For ARM architecture I have a Raspberry PI 3, will see how KVM can be enabled on it's Linux Kernel an if everything works fine, will post also an example with ARM CPU.

Reply:
`Firecracker currently supports physical Linux x86_64 hosts` is mentioned a few lines above; I'd rather hold off ARM instructions in the getting started guide until ARM support is fully in.